QEfficient/cloud/infer.py

```diff
@@ -95,6 +96,7 @@ def main(
         description="Inference command, the model will be downloaded from HF, optmized, compiled, executed on Cloud AI 100"
     )
     parser.add_argument("--model-name", "--model_name", required=True, help="HF Model card name/id")
+    parser.add_argument("--model-path", "--model_path", required=False, help="Path to custom model")
```

Comment:
again let's handle when both `cache_dir` and `model_path` is passed, Also, please explain in help that the `moded_path` should be the path of the `dir` where model weights and config files exist.

Reply:
Let's discuss if we need to handle `cache_dir` and `model_path` separately in the above comment

Will make the Help section for `model_path` more descriptive